exporter/awsemfexporter/grouped_metric.go

```diff
@@ -52,7 +54,25 @@ func addToGroupedMetric(pmd *pdata.Metric, groupedMetrics map[interface{}]*Group
 			continue
 		}
 
+
 		labels := dp.Labels
+
+		if isPod, ok := labels["Type"]; ok {
```

Comment:
nit: Can we rename the variable to `metricType`?

Reply:
oh yeah, sorry I missed this